src/worker.c

```diff
@@ -219,22 +219,24 @@ kore_worker_dispatch_signal(int sig)
 }
 
 void
-kore_worker_privdrop(void)
+kore_worker_privdrop(int id)
```

Comment:
no need, worker 0 when TLS is enabled will always be keymgr process. So you can check worker->id.

Reply:
done